doc/manual/generate-manpage.nix

```diff
@@ -2,7 +2,7 @@ let
   inherit (builtins)
     attrNames attrValues fromJSON listToAttrs mapAttrs groupBy
     concatStringsSep concatMap length lessThan replaceStrings sort;
-  inherit (import ./utils.nix) attrsToList concatStrings optionalString filterAttrs trim squash unique;
+  inherit (import <nix/utils.nix>) attrsToList concatStrings optionalString filterAttrs trim squash unique;
```

Comment:
This implies regular users also get this stuff? If so, I don't think that's good.

Reply:
No, this is only added in `showHelp()` in main.cc

Reply:
Oh phew. Thanks.